engine/src/main/java/org/terasology/math/ChunkMath.java

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright 2013 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.math;
+
+import org.terasology.math.geom.Vector3f;
+import org.terasology.world.chunks.ChunkConstants;
+
+/**
+ * Collection of math functions.
+ *
+ * @author Benjamin Glatzel <benjamin.glatzel@me.com>
```

Comment:
General question rather than specific to this PR: does it still make sense to have these @author lines? It seems to me that git's annotate does a better job anyway, especially as it keeps track of changes. I mean, it's not like begla is going to be primarily responsible for this file any time soon, right?